### ISSUE TITLE
Sending a malformatted aggregation request with CLI does not return a useful message

### ISSUE THREAD

**Issue by fmassot** [quickwit-oss/quickwit#2800]
Example

````
./quickwit index search --index stackoverflow --query "search AND engine" --aggregation '{"foo": {"terms": {"field1": "tags"}}}'
✘ Command failed: Api error: (status=400 Bad Request)

Caused by:
    (status=400 Bad Request)

```

**Comment by fmassot** [comment]
#2843 can fix this issue.

**Comment by fmassot** [comment]
Now it should work as expected, can you check that @evanxg852000 ?

**Comment by PSeitz** [comment]
We still have the serde untagged issue: https://github.com/quickwit-oss/quickwit/issues/1485
There's a PR that adds better error messages for serde untagged, although that's a sub-optimal solution in our case https://github.com/serde-rs/serde/pull/2376

**Comment by fmassot** [comment]
Yes #1485 is still a very annoying issue. I'm closing #2800 was only pointing at the error handling at the REST/CLI level.
I checked that we have at least the error message returned by the CLI.

Just to confirmed that #2843 has fixed this issue but definitely not #1485

```bash
✘ Command failed: Api error: (code=400 Bad Request, message=Invalid aggregation request: data did not match any variant of untagged enum QuickwitAggregations)

Caused by:
    (code=400 Bad Request, message=Invalid aggregation request: data did not match any variant of untagged enum QuickwitAggregations)
```
